opentracing-spring-cloud-starter/src/test/java/io/opentracing/contrib/spring/cloud/NoDepsTest.java

```diff
@@ -50,6 +50,11 @@ public void testNoRxJavaHooks() throws ClassNotFoundException {
     this.getClass().getClassLoader().loadClass("rx.plugins.RxJavaHooks");
   }
 
+  @Test(expected = ClassNotFoundException.class)
+  public void testNoRedisJavaHooks() throws ClassNotFoundException {
+    this.getClass().getClassLoader().loadClass("org.springframework.data.redis.core.RedisTemplate");
```

Comment:
Could you please also add a check on some core redis class?

Reply:
I'm not sure how you mean. I could only depend on some java client class but that would restrict the integrations allowed with spring-data. IMO if someone depends on `spring-data-redis` is because they use Redis, that together with the autoconfig options should be sufficient to activate the aspect(?)

Reply:
Test if redis classes are not on classpath like: `this.getClass().getClassLoader().loadClass("redis.clients.jedis.Client");`

Reply:
If you want to go furthher with tests look at https://github.com/opentracing-contrib/java-spring-cloud/blob/master/opentracing-spring-cloud-starter/pom.xml#L138 profile. This will require adding a dependency on redis spring starter and adding an integration test here https://github.com/opentracing-contrib/java-spring-cloud/tree/master/opentracing-spring-cloud-starter/src/test/java/io/opentracing/contrib/spring/cloud